api_tests/lib/cnd_runner.ts

```diff
@@ -1,7 +1,12 @@
+import * as fs from "fs";
+import { promisify } from "util";
 import { CndInstance } from "../lib_sdk/cnd_instance";
 import { CND_CONFIGS } from "./config";
 import { LedgerConfig } from "./ledger_runner";
 
+const unlinkAsync = promisify(fs.unlink);
```

Comment:
Why not using `fs.existsSync` and `fs.unlinkSync` ?

Reply:
Because they are not async :(